server/src/main/java/org/spine3/server/stand/AggregateStateId.java

```diff
@@ -37,19 +51,33 @@
  * @author Alex Tymchenko
  */
 public final class AggregateStateId<I> {
+
+    private static final AggregateStateIdStringifier stringifier =
+            new AggregateStateIdStringifier();
+
+    static {
+        StringifierRegistry.getInstance()
+                           .register(stringifier(),
+                                     AggregateStateId.class);
+    }
+
     private final I aggregateId;
     private final TypeUrl stateType;
 
     private AggregateStateId(I aggregateId, TypeUrl stateType) {
-        this.aggregateId = aggregateId;
-        this.stateType = stateType;
+        this.aggregateId = checkNotNull(aggregateId);
+        this.stateType = checkNotNull(stateType);
     }
 
     @CheckReturnValue
     public static <I> AggregateStateId of(I aggregateId, TypeUrl stateType) {
         return new AggregateStateId<>(aggregateId, stateType);
     }
 
+    public static Stringifier<AggregateStateId> stringifier() {
```

Comment:
Intuitively a user will search for a `Stringifier` for `AggregateStateId` within the class itself since of other types we declare the `Stringifier`s in the classes themselves or in the related utility classes (e.g. `Events.stringifier()`).